video/vdu_stream_processor.h

```diff
@@ -151,21 +152,58 @@ uint8_t VDUStreamProcessor::readByte_b() {
 	return readByte();
 }
 
-// Read an unsigned word from the serial port (blocking)
+// Read a given number of bytes from the serial port into a buffer
+// Returns number of remaining bytes
+// which should be zero if all bytes were read
+// but will be non-zero if the read timed out
 //
-uint32_t VDUStreamProcessor::readLong_b() {
-	uint32_t result;
-	while(inputStream->available() < sizeof(uint32_t));
-	inputStream->readBytes((uint8_t*)&result, sizeof(uint32_t));
-  	return result;
```

Comment:
readLong is no longer used anywhere - it was previously only being used to load bitmap data into a buffer, and that usage is now replaced by `readIntoBuffer`.  if, in the future, it's functionality is needed then it can quite easily be emulated via a `readIntoBuffer` call.

whilst the new call is slightly more complex, testing has shown that data transfer performance is identical